MANIFEST.in

```diff
@@ -1 +1,9 @@
-# Run check-manifest to generate rules
+include *.py
+include *.rst
+include *.yml
+include .gitlint
+recursive-include docs *.ipynb
```

Comment:
Is it okay that this will include the `test-example` notebook in the release?

Reply:
I could go either way on that one.  I think I'd include it, just because it seems better to include more of the source in the release, if it isn't adding significantly to the file size.  But I think it'd be fine to leave it out as well, seems unlikely anyone would want to look at it.